nowcasting_forecast/models/hub.py

```diff
@@ -0,0 +1,159 @@
+"""
+Originally Taken from https://github.com/rwightman/
+
+https://github.com/rwightman/pytorch-image-models/
+blob/acd6c687fd1c0507128f0ce091829b233c8560b9/timm/models/hub.py
+"""
+
+import json
+import logging
+import os
+from functools import partial
+
+import torch
+
+try:
+    from huggingface_hub import cached_download, hf_hub_url
+
+    cached_download = partial(cached_download, library_name="dgmr")
+except ImportError:
+    hf_hub_url = None
+    cached_download = None
+
+from huggingface_hub import CONFIG_NAME, PYTORCH_WEIGHTS_NAME, ModelHubMixin, hf_hub_download
+
+MODEL_CARD_MARKDOWN = """---
+license: mit
+tags:
+- nowcasting
+- forecasting
+- timeseries
+- remote-sensing
+---
+
+# {model_name}
```

Comment:
do we need to fill this in? I can

Reply:
Probably, it is what shows up on the model card in HF. So more of a description is probably helpful.